unit_test/batched/Test_Batched_SerialAxpy.hpp

```diff
@@ -0,0 +1,147 @@
+/// \author Kim Liegeois (knliege@sandia.gov)
+
+#include "gtest/gtest.h"
+#include "Kokkos_Core.hpp"
+#include "Kokkos_Random.hpp"
+
+//#include "KokkosBatched_Vector.hpp"
+
+#include "KokkosBatched_Axpy_Decl.hpp"
+#include "KokkosBatched_Axpy_Impl.hpp"
```

Comment:
The above comment on Impl is realted to this. We want our tests to link to our library as opposed to include the Impl file.

Reply:
When ETI is on.